# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x86/microcode_ctl/devfs.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN microcode_ctl-1.06/microcode_ctl.c microcode_ctl-1.06-new/microcode_ctl.c
--- microcode_ctl-1.06/microcode_ctl.c	2001-04-15 00:18:13.000000000 +0200
+++ microcode_ctl-1.06-new/microcode_ctl.c	2004-06-20 21:02:05.699653000 +0200
@@ -36,7 +36,7 @@
 #define BUFFER_SIZE	4096
 #define MAX_MICROCODE	2000000
 
-#define MICROCODE_DEVICE_DEFAULT	"/dev/cpu/microcode"
+#define MICROCODE_DEVICE_DEFAULT	"/dev/misc/microcode"
 #define MICROCODE_FILE_DEFAULT		"/etc/microcode.dat"
 
 /* yuck */
